genienlp/util.py

```diff
@@ -566,11 +566,11 @@ def make_data_loader(dataset, numericalizer, batch_size, device=None, train=Fals
     else:
         return data_loader
 
-def ned_dump_entity_type_pairs(split, path, name, utterance_field):
-
-    with open(os.path.join(os.path.dirname(path), f'{name}_labels.jsonl'), 'w') as fout:
-        for ex in split:
-            text = ex.context_plus_question_with_types
+def ned_dump_entity_type_pairs(dataset, path, name, utterance_field, sep_token):
```

Comment:
Do we need to have `sep_token` here? Is it possible to instead of concatenating context and question to get `text`, dump them separately?

Reply:
Yeah, we can do that. I'll push again.